modules/server_new/src/clue/scala/observe/common/ObsQueriesGQL.scala

```diff
@@ -17,9 +17,12 @@ object ObsQueriesGQL:
 
   @GraphQL
   trait ActiveObservationIdsQuery extends GraphQLOperation[ObservationDB]:
+    // TODO The ODB API doesn't provide a way to filter ready observations,
+    // so we filter by accepted proposals for now.
+    // Revise this when the API supports it OR we start getting obersvations from the scheduler.
```

Comment:
Wouldnt the obs workflow have a state saying we are ready?

Reply:
Yes, but:
1) There's no way to set an observation as `Ready` yet.
2) There's no way to filter `Ready` observations.